docs/platform/howto/cleanup-powered-off-services.rst

```diff
@@ -0,0 +1,22 @@
+Periodic cleanup of powered-off services
+========================================
+
+Aiven platform provides a way to power-off services stop accumulating usage costs when not in use. However leaving them powered-off for an extended period of time comes with disadvantages too.
+
+Aiven platform receive regular maintenance updates to keep the services updated with the latest supported versions upstream, including fixes to bugs, possible security vulnerabilities, and overall improvements to performance and memory consumption.
+
+Keeping services in powered-off state for a long time lowers the feasibility of a smooth upgrade path, making it harder for us to continue supporting your needs.
+
+.. note:: 
+    * It is recommended that you regularly review your services and delete those that are no longer needed. This will allow Aiven to focus on supporting the services that you actively use and better utilize platform resources. 
+    * If a service has been powered off for 90 days, you will receive email notifications reminding you that the service has been inactive for a prolonged period. If the service remains powered off for 180 consecutive days, it will be marked for automatic deletion.
```

Comment:
Consider changing the tenses to simple and present ones:

If a service is powered off for 90 days, you receive email notifications reminding you that the service is inactive for a prolonged period. If the service remains powered off for 180 consecutive days, it is subject to automatic deletion.